.github/workflows/basic-tests.yml

```diff
@@ -337,24 +337,24 @@ jobs:
       - name: Install twine
         run: pip install twine
       - name: "Check Airflow create minor branch command"
-        run: |
-          ./scripts/ci/testing/run_breeze_command_with_retries.sh \
-          release-management create-minor-branch --version-branch 2-8 --answer yes
+        run: >
+          breeze release-management create-minor-branch
```

Comment:
I think those retries were really needed when we wanted to run more stuff - let's see if it will work - if not then we will add retries in internal SVN operations.

Reply:
Thanks!  yeah lets see